src/core/css/inc/mixins.scss

```diff
@@ -5,3 +5,15 @@
   box-shadow:0 0 5px 0 $focus-outline-color;
   outline:none;
 }
+
+@mixin respond-to($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    @media #{map-get($breakpoints, $breakpoint)} {
+      @content;
+    }
+  }
```

Comment:
Yeah, it will create a media query where it's used. 

Arranging files by media-query would work, but it's way more of a head-ache to maintain especially considering the minimal usage here.